pkg/apply/applier.go

```diff
@@ -157,21 +157,30 @@ func (a *Applier) Run(ctx context.Context) <-chan event.Event {
 			// give up. Eventually we might be able to determine which errors
 			// are fatal and which might allow us to continue.
 			ch <- event.Event{
-				Type: event.ErrorEventType,
+				Type: event.ErrorType,
 				ErrorEvent: event.ErrorEvent{
 					Err: errors.WrapPrefix(err, "error applying resources", 1),
 				},
 			}
 			return
 		}
+		// If we get there, then all resources have been successfully applied.
```

Comment:
When I am testing it, I see that I get the messages 

**all resources have been applied
prune completed**

even though the pods are not current. What is the actual meaning of this message in user's perspective.

Reply:
You need to provide the `--status` flag to wait for all resources to be Current. I will update the comment.

Currently the printer outputs information for these events, but we could remove that if we want to. I can think of use cases where it would be helpful to show in the UI how we are moving through the sequence of apply->wait for status-> prune.

Reply:
Yes. These messages makes sense with --status flag. Without that flag, we should not have them displayed in my opinion.

Reply:
So I think these messages make sense either way. When we say that something has been applied (in this context), I think we mean that all resources have been created in the APIServer, not that they have all been fully reconciled. As I mentioned in the previous comment, I think of the flow as apply->wait for status->prune.